### PR TITLE
Create First servlet for the site

### PR DESCRIPTION
**Pull request by MathewLG**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <body>
     <div id="introduction">
         <h1 class="title">Hello, friend.</h1>
-        <p>My name is Mathew Lopez, I am from Guadalajara, México and I'm studying computer science student at Tec de Monterrey campus Guadalajara.
```
Comment (nnooney) [inline portfolio/src/main/webapp/index.html]: Did you mean to remove "studying"?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ function addRandomQuote() {
 }
 async function greeting(){
     const responseFromServer = await fetch('/hi');
-    const textFromResponse = await responseFromServer.text();
+    const myArray = await responseFromServer.json();
```
Comment (nnooney) [inline portfolio/src/main/webapp/script.js:39]: Is there a more descriptive name for this variable? Perhaps something like "greetingArray"?

**Review by nnooney — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (MathewLG) [reply portfolio/src/main/webapp/index.html]: Done

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (MathewLG) [reply portfolio/src/main/webapp/script.js]: Done

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -34,3 +34,9 @@ function addRandomQuote() {
   const greetingContainer = document.getElementById('quote-container');
   greetingContainer.innerText = quote;
 }
+async function greeting(){
+    const responseFromServer = await fetch('/hi');
+    const greetingArray = await responseFromServer.json();
+    const greetingContainer = document.getElementById('greeting-container');
+    greetingContainer.innerText = greetingArray[Math.floor(Math.random()*4)];
```
Comment (nnooney) [inline portfolio/src/main/webapp/script.js:41]: Do you always expect the greetingArray to contain 4 elements? I would use greetingArray.length, so you can add more greetings over time :)

**Review by nnooney — CHANGES_REQUESTED** [review]
One more comment (My apologies for not making it earlier, I'm getting used to GitHub's review UI); otherwise everything looks great!

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (MathewLG) [reply portfolio/src/main/webapp/script.js]: No problem, thanks! 
Done.

**Review by nnooney — APPROVED** [review]

**Pull request merged**